src/d_main.c

```diff
@@ -482,8 +482,22 @@ static void D_Display(void)
 
 		if (rendermode != render_none)
 		{
-			F_WipeEndScreen();
-			F_RunWipe(wipedefs[wipedefindex], gamestate != GS_TIMEATTACK);
+			// miru: we can use the mapheaderinfo to change the forced wipe the instant the map loads
+			if (!mapheaderinfo[gamemap-1]->postlevelwipe)
```

Comment:
if (!(mapheaderinfo[gamemap-1]->postlevelwipe) || (mapheaderinfo[gamemap-1]->postlevelwipe > 100)) ???
(and then kill the <100 check line and indentation below